dataset/dictionary/organizations.ts

```diff
@@ -645,6 +653,22 @@ export default [
     pronunciationJa: "しもつきのこ",
     tags: [ "nodkrai", "organization" ],
   },
+  {
+    en: "Hyperborea",
+    ja: "ヒュペルボレイア",
+    zhCN: "亥珀波瑞亚",
+    zhTW: "亥珀波瑞亞",
+    notes: "テイワット大陸の北方にかつて存在していた王国",
+    tags: [ "nodkrai", "organization" ],
+  },
+  {
+    en: "Dephi Pytho",
+    ja: "デルポイ・ピュートーン",
+    zhCN: "德尔斐·皮托呢",
+    zhTW: "德爾斐·皮托呢",
+    notes: "テイワット大陸にかつて存在していた地名",
+    tags: [ "organization" ],
```

Comment:
As this is a location, let's use the `location` tag here.

```suggestion
    tags: [ "location" ],
```